modules/sqlite/integration/db.copy

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * This file is auto-generated and copied from the performance-lab plugin.
+ *
+ * Please don't edit this file directly.
+ *
+ * @package performance-lab
+ */
+
+// Bail early if the SQLite implementation was not located in the performance-lab plugin.
+if ( ! file_exists( '{SQLITE_IMPLEMENTATION_FOLDER_PATH}/wp-includes/sqlite/db.php' ) ) {
+	return;
+}
+
+// Define SQLite constant.
+if ( ! defined( 'DATABASE_TYPE' ) ) {
+	define( 'DATABASE_TYPE', 'sqlite' );
+}
+
+// Require the implementation from the performance-lab plugin.
+require_once '{SQLITE_IMPLEMENTATION_FOLDER_PATH}/wp-includes/sqlite/db.php';
+
+// Activate the performance-lab plugin if it is not already activated.
+add_action(
+	'init',
+	function() {
+		if ( ! function_exists( 'activate_plugin' ) ) {
+			require_once ABSPATH . 'wp-admin/includes/plugin.php';
+		}
+		if ( is_plugin_inactive( '{PERFLAB_PLUGIN}' ) ) {
+			activate_plugin( '{PERFLAB_PLUGIN}' );
+		}
+	}
+);
```

Comment:
> To be fair, nobody should get to this point anyway - activating this module and then deactivating the plugin is not a good idea.

It's not so much about deactivating the plugin... The problem is that when switching databases, the plugin is not active (since the list of active plugins is a db option). That means that if the user switches from MySQL to SQLite, the plugin would appear in their dashboard as inactive, and they'd need to manually activate it. That's why I thought it best to use the `activate_plugin` function.
We could of course just load the main plugin file instead, and that would give users access to the plugin options etc - but in their plugins screen it would still show as inactive 🤔 

You're 100% correct though, it would be completely inefficient to load these admin function on every frontend request. I pushed a tweak to bail early if `PERFLAB_VERSION` is defined before loading the `wp-admin/includes/plugin.php` file, what do you think?

Reply:
@aristath 
> It's not so much about deactivating the plugin... The problem is that when switching databases, the plugin is not active (since the list of active plugins is a db option).

Makes sense. However based on our conversation in https://github.com/WordPress/performance/pull/547#discussion_r1003822665, I would think that we can find a way to address this problem at its root: If we can install the SQLite DB prior to switching, we could also make sure that
* the Performance Lab plugin is immediately active in the SQLite DB as well
* the same modules that are configured in the old DB are active in the SQLite DB

I think the `PERFLAB_VERSION` constant check at least makes it a bit better, but I'd prefer if we could do something like the above.